src/test/java/harness/CRLJanitor.java

```diff
@@ -26,6 +45,18 @@ public class CRLJanitor {
 
   private static final String DEFAULT_CLIENT_NAME = "cli-test";
 
+  // How long Janitor should wait before cleaning test workspaces. It might be useful
+  // to keep these workspaces around long enough to debug, this should be lowered if not.
+  private static final Duration WORKSPACE_TIME_TO_LIVE = Duration.ofHours(48);
+
+  // Map from CLI server to Janitor's identifier for WSM instance.
+  public static final Map<String, String> serverToWsmInstanceIdentifier =
+      ImmutableMap.of("broad-dev", "dev");
```

Comment:
Opt - Hmm, in your Janitor config map, should we switch from `dev` to `broad-dev`, so we don't have to maintain this map?

Or, instead of maintaining 2 maps:
- In Janitor config, instead of map from WSM nickname to WSM url, just have a list of WSM urls
- In test config, put WSM url

Reply:
Hm, good point. It's kind of clunky keeping a bunch of hardcoded strings in sync across multiple repos, keeping a url allowlist in Janitor config would be cleaner but would still prevent calls to arbitrary URLs.
opened https://github.com/DataBiosphere/terra-resource-janitor/pull/140 for this, I'll update this PR after that one's merged